### PR TITLE
Move to shared InteractiveWindow base

### PR DESCRIPTION
**Pull request by paulvanbrenk**
Instead of depending on a fork of the shared interactive window move to consume the VS interactive window as a nuget package. 

This fixes a number of issues #1360, and #1130. And makes it easier to add improvements in the future, e.g. IntelliSense, and use a debug console.

Note: for readability this PR doesn't actually delete the no longer needed code.

### CODE REVIEW

File: Nodejs/Product/Nodejs/NodejsPackage.cs [hunk 1]
```diff
@@ -180,31 +181,26 @@ private void MakeDebuggerContextAvailable()
...
+            if (toolWindowType == Guids.NodejsInteractiveWindow)
             {
-                window = (IReplWindow2)provider.CreateReplWindow(
-                    this.ReplContentType,
-                    Resources.InteractiveWindowTitle,
-                    Guids.TypeScriptLanguageInfo,
-                    NodejsReplEvaluatorProvider.NodeReplId
-                );
+                var model = (IComponentModel)GetService(typeof(SComponentModel));
+                var replProvider = model.GetService<InteractiveWindowProvider>();
```
Comment (amcasey) [inline Nodejs/Product/Nodejs/NodejsPackage.cs]: Consider extracting a method that returns the repl provider.

File: Nodejs/Product/Nodejs/Nodejs.csproj [hunk 2]
```diff
@@ -1196,6 +1191,9 @@
     <PackageReference Include="Microsoft.VisualStudio.Threading">
       <Version>15.5.13-beta</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.VisualStudio.VsInteractiveWindow">
+      <Version>2.3.0</Version>
+    </PackageReference>
```
Comment (amcasey) [inline Nodejs/Product/Nodejs/Nodejs.csproj]: This is roughly equivalent to a packages.json entry?

Comment (paulvanbrenk) [reply Nodejs/Product/Nodejs/Nodejs.csproj]: Yes, the 'new' way to add nuget packages...

File: Nodejs/Product/Nodejs/NodejsPackage.cs (hunk 1 above)
Comment (paulvanbrenk) [reply Nodejs/Product/Nodejs/NodejsPackage.cs]: yes

**Pull request merged**